src/RTCDataChannel.ts

```diff
@@ -79,7 +79,10 @@ export default class RTCDataChannel extends defineCustomEventTarget(...DATA_CHAN
         return this._readyState;
     }
 
-    send(data: string | ArrayBuffer | ArrayBufferView) {
+    send(data: string): void;
```

Comment:
Why the 3 lines?

Reply:
They're [function overloads](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads). It won't affect the transpiled JavaScript at all - I mainly did it so that it'd match the overloads found in the actual DOM types seen [here](https://github.com/microsoft/TypeScript/blob/7c6521e8fb8f24a31f28e00e92838ad1b975fbb3/lib/lib.dom.d.ts#L11605).